docs/src/pages/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
You can actually just create an `index.mdx` in the content folder and it will work! :) No need for "start" or something.

Reply:
or ... rename `start.mdx` to `index.mdx` for that matter 🙈

Reply:
huh? An `index.mdx` or `start.mdx` is exactly what doesn't work with empty routes, and I thought if we stick to a simple landing page we can just create an `index.tsx` in `src/pages`?